setup/05-negative_testing_rule.py

```diff
@@ -1,20 +1,22 @@
 from rflint.common import KeywordRule, SuiteRule, TestRule, ERROR
 
-ALLOWED_KEYWORDS = [
-    "Open Browser To Login Page",
+MUST_KEYWORDS = [
     "Enter Username",
     "Enter Password",
     "Submit Login Form",
     "Verify That Error Page Is Visible"
 ]
 
+SETUP_KEYWORD = "Open Browser To Login Page"
+
 
 class KeywordImplementationRule05(KeywordRule):
     severity = ERROR
 
     def apply(self, keyword):
         report = False
-        if keyword.name == ALLOWED_KEYWORDS[1] or keyword.name == ALLOWED_KEYWORDS[2]:
+        name = keyword.name.lower().title()
```

Comment:
`lower()`:ing and then `title()`:ing is redundant:
```python
>>> 'asda ADSASD As  sA'.title()
'Asda Adsasd As  Sa'
```

Reply:
Changed to use the common `normalize()` function.